src/main/java/pl/targosz/homework/FileReading.java

```diff
@@ -0,0 +1,14 @@
+package pl.targosz.homework;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileReading {
+    public static void main(String[] args) throws IOException {
+        File file = new File("E:\\KURS JAVA\\Workspaceofficial\\JavaRepository\\src\\main\\java\\pl\\targosz\\homework\\resources\\data.txt");
```

Comment:
U mnie to nie zadziała bo nie mam takiej ścieżki. Zapoznaj się proszę z linkiem poniżej 

https://docs.oracle.com/javase/tutorial/essential/io/path.html